### PR TITLE
Docs: update fluentd guide

### PR DESCRIPTION
**Pull request by alexfornuto**
Since the fluentd guide is also the de-facto Teleport Event Plugin guide, I added some context to the front to talk about this, and thread the needle on the local docker-based guide with the additional helm instructions. Also applied some formatting updates.

### CODE REVIEW

File: docs/pages/management/export-audit-events/fluentd.mdx [hunk 1]
```diff
@@ -4,26 +4,56 @@ description: Forwarding events with Fluentd and Teleport event handler
...
+    c-->d
+end
+e(Teleport)
+style e fill: #844cff,color: #ffffff
+e-->c
+ */}
+
+![Fluentd example diagram](../../../img/management/fluentd-demo-dia.png)
+
+You can follow the parameters below for a local proof-of-concept demo, or use any
```
Comment (ptgott) [inline docs/pages/management/export-audit-events/fluentd.mdx:38]:
```suggestion
You can follow the instructions below for a local proof-of-concept demo, or use any
```

File: docs/pages/management/export-audit-events/fluentd.mdx [hunk 2]
```diff
@@ -4,26 +4,56 @@ description: Forwarding events with Fluentd and Teleport event handler
...
+of the additional installation instructions to configure the Teleport Event Hanndler
```
Comment (ptgott) [inline docs/pages/management/export-audit-events/fluentd.mdx:39]: Which additional instructions does this sentence refer to?

Comment (ptgott) [inline docs/pages/management/export-audit-events/fluentd.mdx:39]:
```suggestion
of the additional installation instructions to configure the Teleport Event Handler
```

File: docs/pages/management/export-audit-events/fluentd.mdx [hunk 3]
```diff
@@ -4,26 +4,56 @@ description: Forwarding events with Fluentd and Teleport event handler
...
+guide, we will:
 
 - Set up Teleport's event handler.
 - Forward events with Fluentd.
 
+This guide also serves as an explanation for the Teleport Event Handler plugin,
+using Fluentd as the target service. We'll create a local Docker container as a
+destinatation for the event handler:
+
+{/*
```
Comment (ptgott) [inline docs/pages/management/export-audit-events/fluentd.mdx:17]: Hopefully we can uncomment this soon! 🤞

File: docs/pages/management/export-audit-events/fluentd.mdx (hunk 2 above)
Comment (alexfornuto) [reply docs/pages/management/export-audit-events/fluentd.mdx:39]: There is definitely a better way to say this. Mostly I'm referring to the helm tabs throughout.

It seems like this guide was originally written with local testing in mind, and over time additional instructions have been piecemealed and tacked on without looking at the guide holistically. This is my first pass attempt at trying to explain that.

File: docs/pages/management/export-audit-events/fluentd.mdx [hunk 4]
```diff
@@ -366,16 +406,14 @@ batch = 20
...
 
 [teleport]
 addr = "example.teleport.com:443"
-client_key = "auth.key"
-client_crt = "auth.crt"
-root_cas = "auth.cas"
+identity = "identity"
 ```
 
 To start the event handler, run the following command:
```
Comment (alexfornuto) [inline docs/pages/management/export-audit-events/fluentd.mdx:419]:
```suggestion
To start the Event Handler, run the following command:
```

File: docs/pages/management/export-audit-events/fluentd.mdx [hunk 5]
```diff
@@ -4,26 +4,56 @@ description: Forwarding events with Fluentd and Teleport event handler
...
+e(Teleport)
+style e fill: #844cff,color: #ffffff
+e-->c
+ */}
+
+![Fluentd example diagram](../../../img/management/fluentd-demo-dia.png)
+
+You can follow the instructions below for a local proof-of-concept demo, or use any
+of the additional installation instructions to configure the Teleport Event Handler
+directly into your infrastructure.
```
Comment (ptgott) [inline docs/pages/management/export-audit-events/fluentd.mdx:40]:
```suggestion
to integrate with your infrastructure.
```
nit: I thought "configure ... directly into" sounded slightly off

**Review by ptgott — APPROVED** [review]

File: docs/pages/management/export-audit-events/fluentd.mdx (hunk 5 above)
Comment (alexfornuto) [reply docs/pages/management/export-audit-events/fluentd.mdx:40]: Agreed. Copy edits FTW! You stare at something long enough and you don't see the forest for the trees.

File: docs/pages/management/export-audit-events/fluentd.mdx [hunk 6]
```diff
@@ -148,22 +178,23 @@ $ ls -l
 # -rw------- 1 bob bob     1078 Jul  1 11:14 server.crt
 # -rw------- 1 bob bob     1766 Jul  1 11:14 server.key
 # -rw------- 1 bob bob      260 Jul  1 11:14 teleport-event-handler-role.yaml
+# -rw------- 1 bob bob      260 Jul  1 11:13 teleport-event-handler-v(=teleport.plugin.version=)-darwin-amd64-bin.tar.gz
```
Comment (alexfornuto) [inline docs/pages/management/export-audit-events/fluentd.mdx:181]:
```suggestion
```

File: docs/pages/management/export-audit-events/fluentd.mdx [hunk 7]
```diff
@@ -242,19 +272,26 @@ command.
...
 <TabItem label="Self-Hosted" scope={["oss","enterprise"]}>
 ```code
-$ tctl auth sign --format=tls --user=teleport-event-handler --out=auth
+$ tctl auth sign --user=teleport-event-handler --out=identity
 ```
 
-This command should result in three PEM-encoded files: `auth.crt`, `auth.key`,
-and `auth.cas` (certificate, private key, and CA certs, respectively).
+This command creates one PEM-encoded file, `identity`. The identity file
+includes both TLS and SSH credentials. Your Event Handler plugin uses the SSH
```
Comment (alexfornuto) [inline docs/pages/management/export-audit-events/fluentd.mdx:279]:
```suggestion
includes both TLS and SSH credentials. The Event Handler plugin uses the SSH
```

**Review by mdwn — APPROVED** [review]

**Pull request merged**